adr/ADR-0002-metrics-namespaces.md

```diff
@@ -0,0 +1,80 @@
+---
+num: 2 # allocate an id when the draft is created
+title: ADR template
+status: "Draft" # One of Draft, Accepted, Rejected
+authors:
+  - "ruivieira" # One item for each author, as github id or "firstname lastname"
+tags:
+  - "core,python,service" # e.g. service, python, java, etc
+---
+
+## Title
+
+    Metrics namespaces
+
+## Context and Problem Statement
+
+Currently, the metrics namespace in `explainability-core` consist of a single package (`org.kie.trustyai.explainability.metrics`) with a single class (`FairnessMetrics`).
+As more metrics will be added in the future, it is important to have a clear structure for the metrics namespace. The namespace should be easy to navigate and understand and reflect a general taxonomy of fairness metrics.
+This will also allow extracting common functionality into abstract classes, and make it easier to add new metrics.
+
+## Goals
+
+Refractor `explainability-core` to have a clear structure for the metrics namespace.
+
+## Non-goals
+
+Add new metrics, or change the algorithms for existing ones.
+
+## Current situation
+
+Currently, the metrics namespace in `explainability-core` consist of a single package (`org.kie.trustyai.explainability.metrics`) with a single class (`FairnessMetrics`).
+
+## Proposal
+
+The proposed metrics namespace is as follows:
```

Comment:
Can you please extend the proposal to consider third party metrics? We expect to enable this thanks to [ADR-0001](https://github.com/trustyai-explainability/community/pull/2) so it could be nice to already consider this.

It might probably happen to have different impl of the same algo

Reply:
That's a good point, thanks @danielezonca.
Updated to reflect this.